### ISSUE TITLE
Allow Javadoc-style comment blocks with a Doxyfile variable

### ISSUE THREAD

**Issue by cfriedt** [doxygen/doxygen#6955]
Javadoc allows comments like this
```
/*****************
 *
 *****************/
```
but doxygen does not recognize them.
Instead, the doxygen manual says to do this
```
/*************//**
 *
 ****************/
```
which some users aren't even aware is required. It also behaves poorly with clang-format.

I'm proposing to add a Doxyfile boolean option JAVADOC_BANNER which will default to NO. When set to YES, it will consider the first and second comments above to be equivalent.

**Comment by albert-github** [comment]
I see no reason why the both versions of comment cannot exists simultaneously.
Having an extra configuration is not necessary.

I've just pushed a proposed patch, pull request #6956

**Comment by cfriedt** [comment]
@albert-github Thanks - I was just about to submit a PR as well. It might be useful to get some feedback on both of them.

**Comment by cfriedt** [comment]
Merged in #6957

**Comment by albert-github** [comment]
Code (#6597) has been integrated in master on github (please don't close the issue as this will be done at the moment of a release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.16.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).

**Comment by cfriedt** [comment]
Thanks - I'm happy that I could contribute :-)
